src/main/java/frc/robot/subsystems/PowerPanel.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.PowerDistribution;
+import edu.wpi.first.wpilibj.RobotBase;
+import edu.wpi.first.wpilibj.simulation.PDPSim;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class PowerPanel extends SubsystemBase {
+	private static final PowerDistribution PDH = new PowerDistribution();
```

Comment:
Why did you make this a subsystem instead of a util?

Reply:
Subsystems provide polling. PowerPanel.periodic() is called every tick and can report the ampere draw and update the voltage. A util would need to be called by somebody, so why not the scheduler.